package.json

```diff
@@ -1,11 +0,0 @@
-{
-  "name": "publish_data_beta",
-  "private": true,
-  "dependencies": {},
-  "devDependencies": {
-    "eslint": "^4.0.0",
```

Comment:
Do we no longer need/use eslint?  It seems like a useful thing to use...

Reply:
It wasn't being used previously. Perhaps a separate PR like the one for govuk-lint (or even better, try and get rid of the JS code ;-)